geometry_msgs/PoseStamped start_pose
---
geometry_msgs/PoseArray next_frontier
